spec/board_spec.rb

```diff
@@ -32,4 +32,61 @@
       expect(@board.valid_coordinate?("A22")).to be  false
     end
   end
+
+  describe 'placment validation' do
+    it 'checks before ship placement' do
```

Comment:
We need another line of set-up define the cruiser variable. This is from the IP prior to the source of these expect statements:
`cruiser = Ship.new("Cruiser", 3)`
`submarine = Ship.new("Submarine", 2)`
We can either instantiate these on lines 8.5 and 9 in the `before(:each)` statement thusly
`@cruiser = Ship.new("Cruiser", 3)` 
`@submarine = Ship.new("Submarine", 2)`
and refer to them in future tests by using the @ symbol before the variable name in subsequent tests, or we can instantiate them in each `it` block without the @ symbol.

Reply:
Oh shoot - good catch.  I thought I had caught all of those.  I will add it in now.